lib/main.js

```diff
@@ -37,6 +37,23 @@ function (moment, Router, L, GUI, numeral) {
           }
       }
 
+      var badNodes = []
```

Comment:
The server's purpose is to deal with this. 

On June 6, 2016 2:54:42 PM GMT+02:00, eberhab <notifications@github.com> wrote:
>> @@ -37,6 +37,23 @@ function (moment, Router, L, GUI, numeral) {
>>            }
>>        }
>>  
>> +      var badNodes = []
>
>I can do that, though I still believe this makes some sense here.
>
>Shouldn't a programm protect itself against faulty data and perform
>some checks on the data it receives? At least on the vital parts, so
>that it does not crash in case it receives some faulty information? 
>
>
>---
>You are receiving this because you commented.
>Reply to this email directly or view it on GitHub:
>https://github.com/plumpudding/hopglass/pull/38/files/8618656b1bb4d339648c1838629af9aada6ef4c5#r65884358

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.